conda/cli/main_info.py

```diff
@@ -20,9 +20,10 @@
 from logging import getLogger
 from os.path import exists, expanduser, isfile, join
 from textwrap import wrap
-from typing import Iterable
+from typing import Any, Iterable
 
 from ..deprecations import deprecated
+from ..models.records import PackageRecord
```

Comment:
Let's move this into a type-checking block:

```suggestion
from typing import Any, Iterable, TYPE_CHECKING

from ..deprecations import deprecated

if TYPE_CHECKING:
    from ..models.records import PackageRecord
```